wiki/Game_modifier/Double_Time/id.md

```diff
@@ -0,0 +1,61 @@
+---
+tags:
+  - double time
+  - double-time
+  - DT
+  - mod
+  - game modifier
+---
+
+# Double Time (mod)
+
+![DT mod icon](/wiki/shared/mods/DT.png "Ikon mod Double Time (DT)")
+
+*Untuk melihat daftar keseluruhan dari semua mod, kunjungi: [Game Modifier](/wiki/id/Game_modifier)*\
+*Kunjungi juga: [Nightcore (mod)](/wiki/id/Game_modifier/Nightcore)*
+
+## Tentang
+
+- Singkatan: DT
+- Tipe: Peningkat Kesulitan
+- Pengali skor:
+  - ![][o!s]: 1.12x
```

Comment:
```suggestion
  - ![][osu!]: 1.12x
```

Reply:
need to rename according to https://github.com/ppy/osu-wiki/pull/5245#issuecomment-817430667